cmd/common/registry.go

```diff
@@ -609,6 +675,37 @@ func buildCommandFromGadget(
 	return cmd
 }
 
+func containsColumn(columns []string, column string) bool {
+	for _, c := range columns {
+		if strings.EqualFold(c, column) {
+			return true
+		}
+	}
+	return false
+}
+
+func expandedColumns(col string) []string {
+	if col == "" {
+		return []string{}
+	}
+	switch col {
+	case kubernetesColumnPrefix:
+		return utils.GetKubernetesColumns()
+	case runtimeColumnPrefix:
+		return utils.GetContainerRuntimeColumns()
+	}
+	return []string{col}
+}
+
+func stableSortByPrefix(prefix string, columns []string) {
```

Comment:
Maybe add a `prefix = prefix + "."` first, to prevent columns named `runtimefoo` to be mixed with `runtime.info` accidentally.